desktop/renderer/launcher.js

```diff
@@ -0,0 +1,22 @@
+import {shell, ipcRenderer, remote} from 'electron'
```

Comment:
ah I see, so here is where [this](https://github.com/keybase/client/pull/1530/files#diff-4f1eb51a735628e8047be815d76c8a4dR105) will probably go